src/global/const.h

```diff
@@ -11,7 +11,8 @@
 #define PHD_135 (PHD_45 * 3) // = 24576 = 0x6000
 
 #define MIN_UI_SCALE 0.5f
-#define MAX_UI_SCALE 2.0f
+#define MAX_TEXT_SCALE 2.0f
+#define MAX_BAR_SCALE 1.5f
```

Comment:
Let's make these a `double` by removing the `f `suffix.

Reply:
Let's leave it as double. `READ_FLOAT` is actually ill-formed because it returns doubles:

```c
#define READ_FLOAT(opt, default_value)
    READ_PRIMITIVE(json_object_get_double, opt, default_value)
```

but let's not touch that here.

As for the constants, at first I liked the current way, but you're right, it'll be more consistent to have separate constants for the minimum values so let's go with your suggestion.